src/modules/service-catalog/components/service-catalog-item/CollapsibleDescription.tsx

```diff
@@ -6,6 +6,7 @@ import ChevronDown from "@zendeskgarden/svg-icons/src/16/chevron-down-fill.svg";
 import { useTranslation } from "react-i18next";
 import { getColorV8 } from "@zendeskgarden/react-theming";
 import { XXXL } from "@zendeskgarden/react-typography";
+import ItemThumbnail from "../service-catalog-list/ItemThumbnail";
```

Comment:
The `ItemThumbnail` component is reused both in the `ServiceCatalogList` and `ServiceCatalogItem` component, it is better to put it in a new folder under components, like `components/item-thumbnail`.

Moreover, we usually don't use default exports, so it is better to export it as a named variable for coherence